engine/resource/src/test/test_resource_mounts.cpp

```diff
@@ -29,6 +29,11 @@
 
 const char* MOUNTFILE_PATH = "liveupdate.mounts";
 
+#define DM_SUPPORT_MUTABLE_ARCHIVE
+#if defined(DM_PLATFORM_VENDOR) // Let's deprecated it now, starting with consoles
+    #undef DM_SUPPORT_MUTABLE_ARCHIVE
+#endif
```

Comment:
Instead of fixing writable archives right now (in a format we don't want to support), we have the chance to say that we don't support it on consoles. (We've already deprecated it)